string target_topic
string type_str
string key
---
bool success